addons/remember-collapsed-categories/userscript.js

```diff
@@ -0,0 +1,57 @@
+export default async function ({ console, addon }) {
+  while (true) {
+    const categoryBody = await addon.tab.waitForElement('[id^="category_body_"]', {
+      markAsSeen: true,
+    });
+    console.log(`Found category ${categoryBody.id}`);
+
+    let categoryCollapse = categoryBody.querySelector(".toggle");
+    categoryCollapse.addEventListener("click", (event) => {
+      event.preventDefault();
```

Comment:
What's the purpose of `preventDefault()` here?

Reply:
Probably to prevent scrolling to the top when clicking the buttons (which are actually `<a>` elements with `href="#"`).